packages/@azure/servicebus/data-plane/test/sendSchedule.spec.ts

```diff
@@ -542,44 +542,151 @@ describe("Cancel multiple Scheduled messages", function(): void {
 });
 
 describe("Message validations", function(): void {
+  const longString =
+    "A very very very very very very very very very very very very very very very very very very very very very very very very very long string.";
   afterEach(async () => {
     await afterEachTest();
   });
 
-  it("MessageId validations", async function(): Promise<void> {
+  async function validationTest(msg: any, catchErrorMsg: string): Promise<void> {
+    let errorFlag = false;
     await beforeEachTest(ClientType.PartitionedQueue, ClientType.PartitionedQueue);
     const sender = senderClient.getSender();
-    let errorMessageIdDecimal = false;
-    await sender.send({ body: "", messageId: 1.5 }).catch((err) => {
-      errorMessageIdDecimal =
-        err &&
-        err.message === "'messageId must be a whole integer. Decimal points are not allowed.";
+    await sender.send(msg).catch((err) => {
+      errorFlag = err && err.message === catchErrorMsg;
     });
+    should.equal(errorFlag, true, "errorFlag is not true(Error not thrown as expected)");
```

Comment:
When this check fails, it will not show what the expected error message is.
How about capturing the actual error message and then checking with the expected value?

Reply:
Good idea.